api/v1alpha1/clusterextension_types.go

```diff
@@ -80,14 +82,15 @@ const SourceTypeCatalog = "Catalog"
 type SourceConfig struct {
 	// sourceType is a required reference to the type of install source.
 	//
-	// Allowed values are ["Catalog"]
+	// Allowed values are "Catalog"
 	//
 	// When this field is set to "Catalog", information for determining the appropriate
 	// bundle of content to install will be fetched from ClusterCatalog resources existing
 	// on the cluster. When using the Catalog sourceType, the catalog field must also be set.
 	//
 	// +unionDiscriminator
 	// +kubebuilder:validation:Enum:="Catalog"
+	// +kubebuilder:validation:Required
 	SourceType string `json:"sourceType"`
 
 	// catalog is used to configure how information is sourced from a catalog. This field must be defined when sourceType is set to "Catalog",
```

Comment:
Should probably update this to reflect any updates we make to the CEL validation following my comment on this structs validations